Vagrantfile-standalone

```diff
@@ -0,0 +1,99 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+require 'yaml'
+
+base_dir = File.expand_path(File.dirname(__FILE__))
+conf = YAML.load_file(File.join(base_dir, "vagrant-standalone.yml"))
+groups = YAML.load_file(File.join(base_dir, "ansible-groups.yml"))
+
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = "2"
+Vagrant.require_version ">= 1.7.0"
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+  # if you want to use vagrant-cachier,
+  # please install vagrant-cachier plugin.
+  if Vagrant.has_plugin?("vagrant-cachier")
+    config.cache.enable :apt
+    config.cache.scope = :box
+  end
+
+  # throw error if vagrant-hostmanager not installed
+  unless Vagrant.has_plugin?("vagrant-hostmanager")
+    raise "vagrant-hostmanager plugin not installed"
+  end
+
+  config.vm.box = "capgemini/apollo"
+  config.hostmanager.enabled = true
+  config.hostmanager.manage_host = true
+  config.hostmanager.include_offline = true
+  config.ssh.insert_key = false
+
+  ansible_groups = groups['ansible_groups'];
+  ansible_groups["mesos_masters"] = []
+  ansible_groups["mesos_slaves"] = []
+  master_infos = []
+
+  masters_conf = conf['masters']
+
+  node = {
+    :hostname => "standalone-cluster",
+    :ip => masters_conf['ip'],
+    :mem => masters_conf['mem'],
+    :cpus => masters_conf['cpus'],
+  }
+
+  master_infos.push(node)
+
```

Comment:
I think this would be useful if it would be dynamic but we are assuming as a prerequisite that is going to be only one host so dont think we need  to master_infos.push(node) and then map. We could just  use node or masters_conf, that would become host_conf or cluster_conf

Personally I'd be in favour of making a one dynamic Vagrantfile supporting any number of masters/slaves including the standalone. Then the yaml file with the settings could be set as an environment var. e.g vagrant_cluster=vagrant-standalone.yaml